#65700 "browse" button on www.zshare.net is two times smaller than on desktop version
# Maybe there are more proper fix for different size of button and text entry field, and should be done
# somewhere in widget/src/gtk2...
--- mozilla/layout/style/forms.css.orig	2007-09-16 08:30:19.000000000 +0300
+++ mozilla/layout/style/forms.css	2007-09-17 16:20:11.000000000 +0300
@@ -359,20 +359,21 @@ input[type="file"] > input[type="text"][
   color: inherit;
   font-size: inherit;
   height: inherit;
   letter-spacing: inherit;
 }
 
 /* button part of file selector */
 input[type="file"] > input[type="button"] {
   height: inherit;
   font-size: inherit;
+  height: 24px;
   letter-spacing: inherit;
 }
 
 /* radio buttons */
 input[type="radio"] {
   -moz-appearance: radio-small;
   width: 13px;
   height: 13px;
   margin: 3px 3px 0px 5px;
   padding: 0 !important;
